classes/item.rb

```diff
@@ -2,12 +2,17 @@
 
 class Item
   attr_reader :id, :archived
-  attr_accessor :publish_date
+  attr_accessor :publish_date, :author
 
-  def initialize(id: Rand.random(1.1000), publish_date: Date.today.year, archived: false)
-    @id = id
+  def initialize(publish_date, archived, id: Random.rand)
```

Comment:
Kindly specify the range.
```suggestion
  def initialize(publish_date, archived, id: Random.rand(1..1000))
```